project/src/components/btn-show-more/btn-show-more.tsx

```diff
@@ -1,7 +1,12 @@
+import { useAppDispatch } from '../../hooks';
+import { clickShowMore } from '../../store/action';
```

Comment:
Экшен лучше называть по тому что он делает, а не как обработчик. Например
```suggestion
import { increaseLimit } from '../../store/action';
```